tests/lib/formatters/jslint-xml.js

```diff
@@ -128,7 +128,7 @@ describe("formatter:jslint-xml", function() {
 
         it("should return a string in JSLint XML format with 1 issue in 1 file", function() {
             var result = formatter(code);
-            assert.equal(result, "<?xml version=\"1.0\" encoding=\"utf-8\"?><jslint><file name=\"foo.js\"><issue line=\"5\" char=\"10\" evidence=\"foo\" reason=\"Unexpected &lt;&amp;&quot;&apos;&gt; foo. (foo)\" /></file></jslint>");
+            assert.equal(result, "<?xml version=\"1.0\" encoding=\"utf-8\"?><jslint><file name=\"foo.js\"><issue line=\"5\" char=\"10\" evidence=\"foo\" reason=\"Unexpected &lt;&amp;&quot;&#39;&gt; foo. (foo)\" /></file></jslint>");
```

Comment:
In contrast to [`xml-escape`](https://www.npmjs.com/package/xml-escape), [`_.escape`](https://lodash.com/docs#escape) uses numeric code character entities instead of named ones. Thus the change.